src/leo-server-app/src/models/user.ts

```diff
@@ -1,26 +1,25 @@
 import mongoose from "mongoose";
+import { UserRole } from "../types/user";
 
 const Schema = mongoose.Schema;
 
-enum UserRole {
-  "OPERATOR" = "OPERATOR",
-  "ADMIN" = "ADMIN",
-}
-
-const userSchema = new Schema({
-  email: {
-    type: String,
-    required: true,
-    unique: true,
-    lowercase: true,
-  },
-  role: Object.values(UserRole),
-  satellites: [
-    {
-      type: mongoose.Schema.Types.ObjectId,
-      ref: "Satellite",
+const userSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      unique: true,
+      lowercase: true,
     },
-  ],
-});
+    role: Object.values(UserRole),
+    satellites: [
+      {
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
same comment about the satellite type above

Reply:
yeye